webapp/src/user.tsx

```diff
@@ -5,6 +5,8 @@ import React from 'react'
 
 const UserContext = React.createContext(undefined as IUser|undefined)
 
+const WorkspaceUsersContext = React.createContext(undefined as Array<IUser>|undefined)
```

Comment:
Probably is more useful to store in the context a map of `UserID -> User`, that way we can use that map to complete the user information everywhere (avatar, username, etcetera).

Reply:
Done.